swaybar/bar.c

```diff
@@ -89,6 +89,13 @@ static void mouse_button_notify(struct window *window, int x, int y, uint32_t bu
 	}
 }
 
+static void mouse_scroll_notify(struct window *window, enum scroll_direction direction) {
+	sway_log(L_DEBUG, "Mouse wheel scrolled %s", direction == SCROLL_UP ? "up" : "down");
+
+	const char *workspace_name = direction == SCROLL_UP ? "prev" : "next";
```

Comment:
Btw. One annoying thing with prev/next is that when changing to a workspace on a different output the mouse pointer follows.